src/config.cpp

```diff
@@ -185,6 +185,14 @@ void parse_program_options(int argc, const char** argv)
         .metavar("PID")
         .optional();
 
+    general_options.toggle("as-pre-sudo-user", "Drop root privileges before launching COMMAND.")
```

Comment:
The internal config option is called `drop_root`, the description is "Drop root privileges ...", but the thing is called `--as-pre-sudo-user`. I think that is confusing. Don't differ between "internal" and "external" naming of features. I think `--drop-root(-privileges)` would have been a better name.